### PR TITLE
initial PR

### PR DESCRIPTION
**Pull request by mbukeRepo**
## Initial PR
What I did in this PR:
:heavy_check_mark: Created initial layout of **buildit** .
:heavy_check_mark: Integrated tailwindcss in the application.

### CODE REVIEW

**Review by mutesialine — APPROVED** [review]

File: components/Layout/About/Statistics.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Container from "../../UI/Container";
+import Flex from "../../UI/Flex";
+export default function Statistics() {
+  return (
+    <Container>
+      <div className="">
```
Comment (karemarsy) [inline components/Layout/About/Statistics.js:7]: why there is this empty className?

File: components/Layout/About/Statistics.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Container from "../../UI/Container";
+import Flex from "../../UI/Flex";
+export default function Statistics() {
```
Comment (karemarsy) [inline components/Layout/About/Statistics.js:4]: Class and component names are always singular

File: components/Layout/Home/Projects.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import Flex from "../../UI/Flex";
+import Button from "../../UI/Button";
+import Container from "../../UI/Container";
+
+export default function Projects() {
```
Comment (karemarsy) [inline components/Layout/Home/Projects.js:5]: Class and component names are always singular

File: components/Layout/Home/Socials.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Container from "../../UI/Container";
+import List from "../../UI/List";
+import Section from "../../UI/Section";
+
+export default function Socials() {
```
Comment (karemarsy) [inline components/Layout/Home/Socials.js:6]: Class and component names are always singular

File: components/UI/Form/Textarea.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export default function Textarea() {
```
Comment (karemarsy) [inline components/UI/Form/Textarea.js:3]: Component files should be named in PascalCase

File: components/UI/Logos.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Container from "./Container";
+import Flex from "./Flex";
+
+export default function Logos() {
```
Comment (karemarsy) [inline components/UI/Logos.js:5]: Class and component names are always singular

File: pages/services.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function Services() {
```
Comment (karemarsy) [inline pages/services.js:3]: Page files should be named in kebab-case

File: pages/_document.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
+import { Head, Html, Main, NextScript } from "next/document";
+
+export default function Document() {
```
Comment (karemarsy) [inline pages/_document.js:3]: Page files should be named in kebab-case

File: components/Layout/Home/Project.js [hunk 9]
```diff
@@ -0,0 +1,56 @@
+import Flex from "../../UI/Flex";
+import Button from "../../UI/Button";
+import Container from "../../UI/Container";
+
+export default function projects() {
```
Comment (karemarsy) [inline components/Layout/Home/Project.js:5]: Component files should be named in PascalCase and Class and component names are always singular

**Review by karemarsy — APPROVED** [review]

**Pull request closed without merging**